src/record/import/types/record.ts

```diff
@@ -1,8 +1,9 @@
 import type { OneOf } from "./field";
 
-export type KintoneRecord = {
+export type LocalRecord = {
   data: { [fieldCode: string]: OneOf };
   metadata: {
+    recordIndex: number;
```

Comment:
I thought it would be better that all records have a unique index while whole process.
But you're correct, it makes an extra assumption. I'll reflect your review!